zipkin-lens/src/components/DependenciesPage/DependenciesGraph.jsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015-2019 The OpenZipkin Authors
+ * Copyright 2015-2020 The OpenZipkin Authors
```

Comment:
Thanks for this PR - just checking is lint also checking license header? It looks like it may be doing so differently than maven - instead of worrying about consistency between two checkers it would be good to disable the lint one.

Reply:
Oh, sorry. eslint does not check license header.
However, now, some files are 2019 and some files are 2020. I think it is strange, so I changed all the files because I think it’s just a good opportunity...
If there are problems, I'll revert it :)

Reply:
I think it best to revert even though some files will change later. ex we don't change all headers when the year changes.